capstone-ui/src/components/SearchPage/SearchPage.jsx

```diff
@@ -15,14 +15,13 @@ export default function SearchPage() {
   useEffect(() => {
     axios.get(`https://api-staging.codingcoach.io/mentors`).then((response) => {
       setMentors(response.data);
-      console.log("response: ", response);
     });
   }, []);
 
   const handleSearch = (searchValue) => {
     setSearchInput(searchValue);
     if (searchInput != "") {
-      const filteredData = mentors.filter((mentor) => {
+      const filteredData = mentors.data.filter((mentor) => {
```

Comment:
Good. You are filtering the based on the search term here in the front end. 
The next step is to put the search algorithm in the backend. 
The action of searching should trigger an AJAX request to the server. The server would make a search and would return some results to the front end layer.